src/routes/-components/FooterCTA.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link } from "@tanstack/react-router";
+import rocketIcon from "../../assets/rocket.png";
+import robot from "../../assets/robot-assistant.png";
+import ellipse from "../../assets/ellipse.png";
+export const FooterCTA = () => {
+  return (
+    <div
+      data-test="FooterCTA"
+      className="relative flex flex-col items-center gap-4 p-4 md:gap-10 lg:flex lg:flex-row lg:p-[100px] lg:pl-[20%]"
```

Comment:
consider removing the realtive style to make sure it's properly positioned when scrolled to the end

Reply:
![alt text](image.png)

Reply:
I have corrected this to ensure it scrolls well to the end without the initial empty space